datasets/rd2/config_npsp_for_ldv_data_load.cls

```diff
@@ -0,0 +1,82 @@
+/**
+* @description Configure NPSP Enhanced Recurring Donation Settings for bulk loading via SnowFakery.
+* This script assumes that the all NPSP Settings objects have been properly seeded with default values
+* either by loading the NPSP Settings page or running the default_npsp_settings cci task.
+* @author Michael Smith
+* @date 2020-08-11
+*/
+
+/**
+ * @description Before the data load:
+ * - Validate the RD2 is enabled in the org
+ * - Set "First Installment" to never to allow bulk data loading without impacting cpu
+ * - Configure automatic naming so that all new RD's are named nicely
+ * - Mock the migration just to avoid any validation errors
+ * - Configuring the Household Naming format as "{Lastname} {FirstName} Household" so that it's easier to.
+ *   read when there are duplicate households with the same last name.
+ * - Disable Household Address auto creation
+ */
+public static void before_data_load() {
```

Comment:
I will add a Work Item to the backlog to have our `load_custom_settings` task support namespace injection, which would obviate the need for Anonymous Apex classes like this.